### PR TITLE
Update watchers.rst

### PR DESCRIPTION
**Pull request by chief-wrench**
Added fresher repo for the sublimetext watcher

### CODE REVIEW

**Comment by chief-wrench** [comment]
This is the first time I have contributed anything to a repo of someone else. Please be gentle...

**Comment by johan-bjareholt** [comment]
Thank you for contributing, looks good!

I don't use sublime so I don't know much about those watchers, but you seem to be correct that the previously linked watcher is unmaintained and slightly buggy. @prplecake created an issue without any response from @kostasdizas in the middle of January and never got a response and it seems like he made a lot of improvements to the watchers since then. @kostasdizas also has not had any activity at all on GitHub since September 2020.

I just found it to be a shame that the maintained watcher is not in packagecontrol, but it would also create a lot of confusion if there were two watchers there.

In short, I wish that @prplecake wouldn't have had to fork the project in the first place and that they could work together.

**Comment by kostasdizas** [comment]
Hello @johan-bjareholt, I created the watcher last year, but in recent months I haven't had time to look after the reported issues. I did assign the issue to myself as a sign of progress, but unfortunately I have not fixed it yet.
I wasn't even aware of the further changes because @prplecake seems to have copied my work, did not fork the repository, and did not get in touch at all or attempted to create a pull request with his changes.

**Comment by prplecake** [comment]
I assumed the repo was unmaintained since I didn’t get a response to my issue. 

Past the watcher not getting it’s settings on launch, I made additional improvements (especially on the windows side) since AW expects Unix-style paths.

**Comment by chief-wrench** [comment]
Hi everyone, maybe it would be a good idea to merge these two versions and commit the result to the main aw-repo instead? It would be the logical place to look for a n00b like me.

**Comment by johan-bjareholt** [comment]
@chief-wrench I agree. It just seems like a communication issue.

Would you both be open to working together, either one one of your existing repos or in an official repo under the activitywatch org?

**Comment by prplecake** [comment]
I would be amicable to such an endeavor.

**Comment by kostasdizas** [comment]
The project is always open to PRs and would gladly work with anyone. What I definitely do not appreciate is my work being illegally stolen. 

@johan-bjareholt if you feel like moving the repo under the activiywatch org would be better, I'm happy to do that.
